assets/schema_database.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/taxprofiler/master/assets/schema_database.json",
+    "title": "nf-core/taxprofiler pipeline - params.database schema",
+    "description": "Schema for the file provided with params.database",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "tool": {
+                "type": "string",
+                "exists": true,
+                "pattern": "^\\S+$",
+                "enum": [
+                    "bracken",
+                    "centrifuge",
+                    "diamond",
+                    "ganon",
+                    "kaiju",
+                    "kmcp",
+                    "kraken2",
+                    "krakenuniq",
+                    "malt",
+                    "metaphlan",
+                    "motus"
+                ],
+                "errorMessage": "Invalid tool name. Please see documentation for all supported profilers. Currently these classifers are included: bracken, centrifuge, diamond, ganon, kaiju, kmcp, kraken2, krakenuniq, malt, metaphlan, motus.",
+                "meta": ["tool"]
+            },
+            "db_name": {
+                "type": "string",
+                "exists": true,
+                "pattern": "^\\S+$",
+                "errorMessage": "The unique name of the database should be provided.",
+                "meta": ["db_name"]
+            },
+            "db_params": {
+                "type": "string",
+                "pattern": "^[^\"']*$",
+                "anyOf": [
+                    {
+                        "properties": {
+                            "tool": { "const": "bracken" }
+                        },
+                        "not": {
+                            "pattern": ".*;"
+                        },
+                        "errorMessage": "Invalid database db_params entry. Bracken requires a semi-colon if passing parameter."
+                    },
+                    {
+                        "properties": {
+                            "tool": { "const": "kmcp" }
+                        },
+                        "pattern": ".*;$",
+                        "errorMessage": "Invalid database db_params entry. KMCP only requires a semi-colon if passing arguments to KMCP profile, in cases of which the arguments should go after the semi-colon."
+                    },
+                    {
+                        "not": {
+                            "properties": {
+                                "tool": { "enum": ["bracken", "kmcp"] }
+                            }
+                        },
+                        "errorMessage": "Invalid database db_params entry."
```

Comment:
```suggestion
                        "errorMessage": "Invalid database `db_params` entry."
```

Reply:
This message doesn't really say anything to help the user debug why it's an invalid database entry, can this be made a bit clearer?

Reply:
@nvnieuwk, for the `db_params` of all tools, quotes are generally not allowed. However, there are additional restrictions for `bracken` and `kmcp`. Should I move the `errorMessage` from line 66 to line 62?
So line 62 would be: 
```
"errorMessage": "Invalid database db_params entry. No quotes allowed."
```